build-logic/convention/src/main/kotlin/AndroidCommonConventionPlugin.kt

```diff
@@ -61,10 +61,16 @@ class AndroidCommonConventionPlugin : Plugin<Project> {
                 }
 
                 lint {
-                    abortOnError = false
```

Comment:
With this the `lint` tasks were ending with exit code 0 and because of that the CI was just not doing anything.